libs/game/spriteset.ts

```diff
@@ -1,7 +1,7 @@
 /* 
 * A set of sprites 
 */
-class SpriteSet {
+export class SpriteSet {
```

Comment:
Hmm strange but this solves the python compiler errors, and everything compiles and runs fine. Maybe it's how we scrape for APIs or something. Any ideas?

Reply:
Actually, there is an error, I just didn't see it because other errors where in front.

> Yeah, we probably don't export it even though we should. I suggest moving it to some namespace though.

sounds good, I'll move it into the sprite namespace